### ISSUE TITLE
Too messy project files

### ISSUE THREAD

**Issue by meakcey** [nikolaseu/neuvision#18]
It seems you really had done well project but it is nested too much.
It is really hard to follow.
Project inside project inside project and goes on.
Do you have any simplified version for multi camera calibration?

**Comment by nikolaseu** [comment]
Hi!

I think it's pretty well organized, but I agree it's probably split too much and some libs could be joined into one. Keep in mind that a lot of `.pro` files are for the different libs, so I can use them easily in different apps.

The only project you need to open for multi camera calibration is https://github.com/nikolaseu/neuvision/blob/master/Z3DMultiCameraCalibration.pro, you will have everything visible there.

If you want to compile & run the app you need to update the paths to opencv include and libs folder, like in https://github.com/nikolaseu/neuvision/blob/master/3rdparty/opencv.pri#L8, and that should be enough. If you don't have opencv binary files in your path, you will also have to add them, or copy them right next to the generated executable, in the `bin` folder.
